proposals/memory64/Overview.md

```diff
@@ -141,6 +141,63 @@ have to support 32-bit memory addresses in their ABI.
       ```
   - (and similar for memory instructions from other proposals)
 
+* The [SIMD proposal][simd] extends `t.load memarg` and `t.store memarg`
+  above such that `t` may now also be `v128`, which accesses a 16-byte quantity
+  in memory that is also 16-byte aligned.
+
+  In addition to this, it also has these SIMD specific memory operations (see
+  [SIMD proposal][simd] for full semantics):
+  - `v128.loadN_zero memarg` (where N = 32/64):
+    Load a single 32-bit or 64-bit element into the lowest bits of a `v128`
+    vector, and initialize all other bits of the `v128` vector to zero.
+  - `v128.loadN_splat memarg` (where N = 8/16/32/64):
+    Load a single element and splat to all lanes of a `v128` vector. The natural
+    alignment is the size of the element loaded.
+  - `v128.loadN_lane memarg v128 immlaneidx` (where N = 8/16/32/64):
+    Load a single element from `memarg` into the lane of the `v128` specified in
+    the immediate mode operand `immlaneidx`. The values of all other lanes of
+    the `v128` are bypassed as is.
+  - `v128.storeN_lane memarg v128 immlaneidx` (where N = 8/16/32/64):
+    Store into `memarg` the lane of `v128` specified in the immediate mode
+    operand `immlaneidx`.
+  - `v128.loadL_sx memarg` (where L is `8x8`/`16x4`/`32x2`, and sx is `s`/`u`):
+    Fetch consecutive integers up to 32-bit wide and produce a vector with lanes
+    up to 64 bits. The natural alignment is 8 bytes.
+
+  All these operations now take 64-bit address operands when used with a
+  64-bit memory.
+
+* The [Threads proposal][threads] has `atomic` versions of `t.load`, `t.store`,
+  `t.loadN_sx` and `t.storeN_sx` specified above, except with `.` replaced by
```

Comment:
Threads only has zero-extending atomic loads, but not sign-extending loads (instead it added register sign-extension operators which were broken out into their own proposal).

Reply:
Thanks, fixed.